documents-webapp/src/main/webapp/vue-app/documents/components/body/table/DocumentInfoDrawer.vue

```diff
@@ -236,8 +238,10 @@ export default {
       this.$documentFileService.updateDescription(ownerId,this.file)
         .then(() => {
           this.displayAlert(this.$t('documents.alert.success.description.updated'));
-          this.showDescription = true;
+          this.showDescription = this.file.description && this.file.description.length ? true : false;
+          this.showNoDescription = this.file.description && this.file.description.length ? false : true;
```

Comment:
Do we really need the variable this.showNoDescription ? isn't is just the negation of this.showDescription ?
```suggestion
          this.showDescription = this.file.description && this.file.description.length;
          this.showNoDescription = !this.showDescription;
```